docs/concepts/jcloud/yaml-spec.md

```diff
@@ -414,6 +415,26 @@ executors:
     uses: jinaai+docker://<username>/Executor1
 ```
 
+Executor level `labels`:
+```{code-block} yaml
+---
+emphasize-lines: 9-12
```

Comment:
This is not correct. Please count the lines you want to highlight.

Reply:
updated the file, but line numbers seem correct. Please check once in the new commit.